src/utils.py

```diff
@@ -0,0 +1,61 @@
+# Save a dictionary into a pickle file.
+import pickle
+import os
+import pandas as pd
+import json
+
+
+class SaveRead():
```

Comment:
you don't need this class. everything is static here anyway, so it might as well be pulled out.

Reply:
That's also a preference I would think. 
It helps encapsulate similar methods under the same class. 
But maybe no need? What would you do?

Reply:
you've already collected them to the same file, so that's enough for python.
I'd do something similar to you in java, but that's just because in java I would have no choice.

Reply:
bottom line I would not use a class\object unless I need to be stateful or make subdivision within a file.